dev/import-beats/streams.go

```diff
@@ -232,3 +263,31 @@ func isConfigEntryRelatedToMetricset(entry mapStr, moduleName, datasetName strin
 	}
 	return metricsetRelated, nil
 }
+
+// determineInputVariableType method determines the most appropriate type of the value or the value in array.
+// Support types: text, password, bool, integer
+func determineInputVariableType(name, v interface{}) string {
+	if arr, isArray := v.([]interface{}); isArray {
+		if len(arr) == 0 {
+			return "text" // array doesn't contain any items, assuming default type
+		}
+		return determineInputVariableType(name, arr[0])
+	}
+
+	if _, isBool := v.(bool); isBool {
+		return "bool"
+	} else if _, isInt := v.(int); isInt {
+		return "integer"
+	}
+
+	if name == "password" {
+		return "password"
+	}
+	return "text"
+}
+
+func toVariableTitle(name string) string {
+	name = strings.ReplaceAll(name, "_", " ")
```

Comment:
Nice trick :-)

Reply:
> We should make sure in our package checklist somewhere is to check it looks good in the UI.

https://github.com/elastic/package-registry/blob/master/CONTRIBUTING.md
Point no 7: _Review titles and descriptions in manifest files_